cfgov/jinja2/v1/_includes/molecules/related-metadata.html

```diff
@@ -135,13 +140,9 @@ <h3 class="h4">
    ========================================================================== #}
 
 {% macro _text(text) %}
-
-<p class="u-mb0">
     {{ parse_links(text.blob) | safe }}
-</p>
 {% endmacro %}
 
-{# TODO: This function is not used and can be removed. #}
```

Comment:
While `_topics` is called, I'm unclear how `{{ _list( related_metadata_tags(page), true ) }}` within is called.